RangeSlider: update demo description (#2064)
DevExpress/devextreme-demos: JSDemos/Demos/RangeSlider/Overview/Angular/description.md

Use the [showRange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#showRange) property to specify if the selected range should be highlighted. You can also use the [step](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#step) property to specify the value change step for the RangeSlider.

If you want to disable the RangeSlider, set the [disabled](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#disabled) property to `true`.

## Handle the Value Change Event

To handle value changes, use two-way binding to bind the [value](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#value) properties of [NumberBox](/Documentation/ApiReference/UI_Components/dxNumberBox/) components to the [start](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#start) and [end](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#end) properties of the RangeSlider.

> The [callValueChange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#callValueChange) property allows you to choose when to change the RangeSlider value and supports two modes: `onMoving` and `onMovingComplete`. You can see the property effect in the **Process Value Changes** section. The first RangeSlider changes its value every time a user slides the handle. The second RangeSlider changes its value only on handle release.

Reviewer: ```suggestion
The [callValueChange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#callValueChange) property allows you to choose when to change the RangeSlider value and supports the `onMoving` and `onMovingComplete` modes. You can see the property's effect in the **Process Value Changes** section. The first RangeSlider changes its value every time a user slides the handle. The second RangeSlider changes its value only when the handle is released.
```

> The [callValueChange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#callValueChange) property allows you to choose when to change the RangeSlider value and supports two modes: `onMoving` and `onMovingComplete`. You can see the property effect in the **Process Value Changes** section. The first RangeSlider changes its value every time a user slides the handle. The second RangeSlider changes its value only on handle release.

Reviewer 2: ```suggestion
The [callValueChange](/Documentation/ApiReference/UI_Components/dxRangeSlider/Configuration/#callValueChange) property allows you to choose when to change the RangeSlider value and supports two modes: `onMoving` and `onMovingComplete`. You can see the property's effect in the **Process Value Changes** section. The first RangeSlider changes its value every time a user slides the handle. The second RangeSlider changes its value only on handle release.
```
